### PR TITLE
Revert ambuzol recipe change

### PR DESCRIPTION
**Pull request by stellar-novas**
## About the PR

Reverts the change to make Ambuzol require ZombieBlood and AmbuzolPlus to require Omnizine.

## Why / Balance

The change was made upstream as a counter to powergaming by making Ambuzol early on, before any sign of zombies. This is not an issue on Delta-V, and most zombie rounds currently end with a crew loss, this change makes it even more difficult for the crew.

## Technical details

N/A

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

N/A

**Changelog**

:cl:
- tweak: Upstream Ambuzol recipe changes have been reverted.

### CODE REVIEW

File: Resources/Prototypes/Recipes/Reactions/medicine.yml [hunk 1]
```diff
@@ -213,7 +213,7 @@
       amount: 1
     Ammonia:
       amount: 1
-    ZombieBlood:
+    Blood:
```
Comment (Adrian16199) [inline Resources/Prototypes/Recipes/Reactions/medicine.yml:219]:
```suggestion
    Blood:  # Delta V - Reverts "ambuzol beef 25119"
```

File: Resources/Prototypes/Recipes/Reactions/medicine.yml [hunk 2]
```diff
@@ -223,8 +223,8 @@
   reactants:
     Ambuzol:
       amount: 5
-    Omnizine:
-      amount: 5
+    ZombieBlood:
```
Comment (Adrian16199) [inline Resources/Prototypes/Recipes/Reactions/medicine.yml:229]:
```suggestion
    ZombieBlood:  # Delta V - Reverts "ambuzol beef 25119"
```

**Comment by stellar-novas** [comment]
Guh I forgot these were upstream files, I'll go back and add markers

**Comment by stellar-novas** [comment]
Added a marker, though is it needed? This is equivalent to never merging the commit in the first place, and markers are not added for those.

**Comment by Adrian16199** [comment]
> Added a marker, though is it needed? This is equivalent to never merging the commit in the first place, and markers are not added for those.

I mean, adding a note helps a merger or anyone lookin there and see:
Oh wait, this is different compared to wizden!
Instead of:
What a weird thing, I dont recall this being in wizden but no note regardin it.

File: Resources/Prototypes/Recipes/Reactions/medicine.yml [hunk 3]
```diff
@@ -206,14 +206,18 @@
   products:
     Phalanximine: 3
 
+# DeltaV - Reverts Ambuzol recipe changes - Ambuzol and AmbuzolPlus were changed upstream to require
+# ZombieBlood and Omnizine respectively to combat power gaming. This change was irrelevant for Delta-V.
+# This code is licensed under AGPLv3. See LICENSE
```
Comment (DebugOk) [inline Resources/Prototypes/Recipes/Reactions/medicine.yml]: No point in having this here

**Review by DebugOk — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Recipes/Reactions/medicine.yml (hunk 3 above)
Comment (DebugOk) [reply Resources/Prototypes/Recipes/Reactions/medicine.yml]: Referring to line 211

Comment (stellar-novas) [reply Resources/Prototypes/Recipes/Reactions/medicine.yml]: Removed it

**Pull request closed without merging**